### PR TITLE
[cli] Add telemetry tracking for `vercel domains add`

### PR DESCRIPTION
**Pull request by erikareads**
- **[cli] Add telemetry for `vercel domains add`**
- **add changeset**

### CODE REVIEW

File: packages/cli/test/unit/commands/domains/add.test.ts [hunk 1]
```diff
@@ -1,9 +1,66 @@
...
 
-describe.todo('domains add', () => {
-  describe.todo('[name]', () => {
-    describe.todo('[project]', () => {
-      describe.todo('--force');
+describe('domains add', () => {
+  describe('[name]', () => {
+    describe('[project]', () => {
+      describe('--force', () => {
+        it('tracks telemetry data, async', async () => {
```
Comment (TooTallNate) [inline packages/cli/test/unit/commands/domains/add.test.ts:12]: What does "async" mean here?

File: packages/cli/src/commands/domains/add.ts [hunk 2]
```diff
@@ -25,6 +26,14 @@ export default async function add(
...
 ) {
+  const { telemetryEventStore } = client;
+  const telemetry = new DomainsAddTelemetryClient({
+    opts: {
+      store: telemetryEventStore,
+    },
+  });
+
+  telemetry.trackCliFlagForce(opts['--force']);
   const force = opts['--force'];
```
Comment (TooTallNate) [inline packages/cli/src/commands/domains/add.ts]: Swapping the order of these two lines would help (slightly) with minification.

```suggestion
  const force = opts['--force'];
  telemetry.trackCliFlagForce(force);
```

**Review by TooTallNate — DISMISSED** [review]
Minor nit and a question. Nothing blocking though.

File: packages/cli/test/unit/commands/domains/add.test.ts (hunk 1 above)
Comment (erikareads) [reply packages/cli/test/unit/commands/domains/add.test.ts:12]: I copied that from another `vercel domains` telemetry PR that @trek wrote: https://github.com/vercel/vercel/blob/d1743d1757f51ff94f72b714ff048584c9abcd31/packages/cli/test/unit/commands/domains/ls.test.ts#L43

Comment (trek) [reply packages/cli/test/unit/commands/domains/add.test.ts:12]: Yeah, that was just some moving-too-fast-in-my-editor junk

**Review by trek — APPROVED** [review]

File: packages/cli/test/unit/commands/domains/add.test.ts (hunk 1 above)
Comment (trek) [reply packages/cli/test/unit/commands/domains/add.test.ts:12]: Approved to keep us going but @erikareads or I should swing back and fix both of these after.

Comment (trek) [reply packages/cli/test/unit/commands/domains/add.test.ts:12]: https://github.com/vercel/vercel/pull/12416

**Pull request merged**